x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/rule_exceptions/auto_populate_with_alert_data.cy.ts

```diff
@@ -5,141 +5,64 @@
-import { createRule } from '../../../tasks/api_calls/rules';
-import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { LOADING_INDICATOR } from '../../../../screens/security_header';
+import { getEndpointRule } from '../../../../objects/rule';
+import { createRule } from '../../../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../../../tasks/alerts_detection_rules';
 import {
   addExceptionFromFirstAlert,
   expandFirstAlert,
-  goToClosedAlertsOnRuleDetailsPage,
-  goToOpenedAlertsOnRuleDetailsPage,
   openAddRuleExceptionFromAlertActionButton,
-} from '../../../tasks/alerts';
+} from '../../../../tasks/alerts';
 import {
   addExceptionEntryFieldValue,
   addExceptionEntryFieldValueValue,
-  addExceptionEntryOperatorValue,
   addExceptionFlyoutItemName,
-  selectBulkCloseAlerts,
   submitNewExceptionItem,
-  validateExceptionItemFirstAffectedRuleNameInRulePage,
-  validateExceptionItemAffectsTheCorrectRulesInRulePage,
   validateExceptionConditionField,
   validateExceptionCommentCountAndText,
   editExceptionFlyoutItemName,
   validateHighlightedFieldsPopulatedAsExceptionConditions,
   validateEmptyExceptionConditionField,
-} from '../../../tasks/exceptions';
-import { login, visitWithoutDateRange } from '../../../tasks/login';
-import {
-  goToAlertsTab,
-  goToExceptionsTab,
-  removeException,
-  waitForTheRuleToBeExecuted,
-} from '../../../tasks/rule_details';
+} from '../../../../tasks/exceptions';
+import { login, visitWithoutDateRange } from '../../../../tasks/login';
+import { goToExceptionsTab } from '../../../../tasks/rule_details';
 
-import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
-import { postDataView, deleteAlertsAndRules } from '../../../tasks/common';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../../urls/navigation';
+import { deleteAlertsAndRules } from '../../../../tasks/common';
 import {
   ADD_AND_BTN,
   ENTRY_DELETE_BTN,
   EXCEPTION_CARD_ITEM_CONDITIONS,
   EXCEPTION_CARD_ITEM_NAME,
   EXCEPTION_ITEM_VIEWER_CONTAINER,
-  NO_EXCEPTIONS_EXIST_PROMPT,
-} from '../../../screens/exceptions';
-import { waitForAlertsToPopulate } from '../../../tasks/create_new_rule';
-
-const loadEndpointRuleAndAlerts = () => {
-  cy.task('esArchiverLoad', 'endpoint');
-  login();
-  createRule(getEndpointRule());
-  visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
-  goToRuleDetails();
-  waitForAlertsToPopulate();
-};
+} from '../../../../screens/exceptions';
+import { waitForAlertsToPopulate } from '../../../../tasks/create_new_rule';
 
-describe('Rule Exceptions workflows from Alert', () => {
-  const EXPECTED_NUMBER_OF_ALERTS = '1 alert';
+describe('Auto populate exception with Alert data', () => {
   const ITEM_NAME = 'Sample Exception Item';
   const ITEM_NAME_EDIT = 'Sample Exception Item Edit';
   const ADDITIONAL_ENTRY = 'host.hostname';
-  const newRule = getNewRule();
 
   beforeEach(() => {
+    cy.task('esArchiverUnload', 'endpoint');
```

Comment:
is in it the same what we do in `afterEach`?

Reply:
Same as https://github.com/elastic/kibana/pull/161751#discussion_r1277856316